src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -103,13 +105,76 @@ class RecallControllerTest {
+  private val inAssessmentRecall = Recall(::RecallId.random(), NomsNumber("tr42412"), createdByUserId, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), licenceNameCategory = NameFormatCategory.FIRST_MIDDLE_LAST, bookedByUserId = bookedByUserId, assignee = assignee)
+  private val stoppedRecall = Recall(::RecallId.random(), NomsNumber("ew53523"), createdByUserId, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), licenceNameCategory = NameFormatCategory.FIRST_MIDDLE_LAST, bookedByUserId = bookedByUserId, agreeWithRecall = AgreeWithRecall.NO_STOP)
+  private val recallNotificationIssuedRecall = Recall(::RecallId.random(), NomsNumber("as241245"), createdByUserId, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), licenceNameCategory = NameFormatCategory.FIRST_MIDDLE_LAST, recallNotificationEmailSentDateTime = now)
+  private val dossierInProgressRecall = Recall(::RecallId.random(), NomsNumber("bf54366"), createdByUserId, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), licenceNameCategory = NameFormatCategory.FIRST_MIDDLE_LAST, recallNotificationEmailSentDateTime = now, assignee = assignee)
+  private val dossierIssuedRecall = Recall(::RecallId.random(), NomsNumber("gf35252"), createdByUserId, now, FirstName("Barrie"), MiddleNames("Barnie"), LastName("Badger"), licenceNameCategory = NameFormatCategory.FIRST_MIDDLE_LAST, dossierCreatedByUserId = dossierCreatedByUserId)
+
   @Test
-  fun `gets all recalls`() {
-    every { recallRepository.findAll() } returns listOf(recall)
+  fun `gets all recalls for a band FOUR_PLUS returns all recalls`() {
+    val bearerToken = "Bearer header.payload"
+    every { recallRepository.findAll() } returns listOf(beingBookedOnRecall, bookedOnRecall, inAssessmentRecall, stoppedRecall, recallNotificationIssuedRecall, dossierInProgressRecall, dossierIssuedRecall)
+    every { tokenExtractor.getTokenFromHeader(bearerToken) } returns Token(::UserId.random().toString())
+    val userDetails = mockk<UserDetails>()
+    every { userDetailsService.get(any()) } returns userDetails
+    every { userDetailsService.find(any()) } returns userDetails
+    every { userDetails.caseworkerBand } returns CaseworkerBand.FOUR_PLUS
+    every { userDetails.fullName() } returns "Mickey Mouse"
+
+    val results = underTest.findAll(bearerToken)
+
+    assertThat(results.size, equalTo(7))
 
-    val results = underTest.findAll()
+    assertThat(
+      results,
+      List<RecallResponse>::containsAll,
+      listOf(
+        RecallResponse(beingBookedOnRecall.recallId(), beingBookedOnRecall.nomsNumber, createdByUserId, now, now, FirstName("Barrie"), null, LastName("Badger"), NameFormatCategory.FIRST_LAST, Status.BEING_BOOKED_ON),
```

Comment:
Building `expected` like this makes it pretty hard to read - "to see the wood for the trees".
Can we e.g. move the duplication to one or more helper methods in this test please?